### DISCUSSION TITLE
JSON File

### DISCUSSION THREAD

**Discussion by loliveirawebdev** [lowdefy/lowdefy#1359] [Questions and Help] [answered]
Hi everyone.

I'm one of who prefer JSON instead of YAML but the only way that I've found to do it is writing JSON code into the **lowdefy.yaml** file, but i loose a lot of IDE facilities because of that.

Is there any explicit motivation for this?

**Comment by loliveirawebdev** [comment, marked as answer]
lol
i found in the docs that we need to do is this way only in lowdefy.yaml, but other files can be referenced as .json files

sorry, it was not very clear... maybe is good to add this information in the Introduction as well

**Reply by Gervwyk** [reply]
Hi @lalvesoliveira 
Sorry for the delayed response. Yes. That's how it's currently set up. Better docs for this does make sense :) 
You can also _ref other files like .html or .md etc. but the root entry point is the lowdefy.yaml, at least for now.
